tests/src/main/resources/config/local-ibm-fhir.json

```diff
@@ -0,0 +1,8 @@
+{
+    "@class": "com.ibm.cohort.engine.IBMFhirServerConfig",
+    "endpoint": "https://localhost:9443/fhir-server/api/v4",
+    "password": "change-password",
+    "tenantId": "test-fvt",
```

Comment:
Do we have any reason to be concerned about this being published?

Reply:
Corey says those are well-known values published in the IBM FHIR user guide.

Reply:
Here are the docs: https://ibm.github.io/FHIR/guides/FHIRServerUsersGuide